app/components/common/ApiJsonFactory.js

```diff
@@ -6,13 +6,17 @@ var commonFactories = angular.module('openevent');
 
 commonFactories.factory('ApiJsonFactory', ['$q', '$http', function ($q, $http) {
 var baseUrl = (config.use_testApi?'testapi/':config.apiBaseGetUrl)
-                + 'event/' + config.eventId + '/';
+                + 'event/' + config.eventId;
 
         return {
             getJson: function ($apiEndpoint) {
-                var deferred = $q.defer(),
-                    httpPromise = $http.get(baseUrl + $apiEndpoint);
-
+                if (($apiEndpoint === "event") && (!config.use_testApi)) {
```

Comment:
Mixed double and single quotes.<br>Identifier 'use_testApi' is not in camel case.<br>'config' is not defined.